extensions/2.0/Vendor/EXT_pbr_attributes/Schema/glTF.EXT_pbr_attributes.schema.json

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "EXT_pbr_attributes glTF extension",
+    "type": "object",
+    "description": "glTF extension that defines the metalness-roughness material model from Physically-Based Rendering (PBR) as a per vertex attribute.",
+    "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+    "properties": {
+        "baseColorAttribSpace": {
+            "type": "string",
+            "enum": ["sRBG", "linear"],
```

Comment:
typo: sRBG -> sRGB I presume :)

Reply:
:-) Yeah, will need to edit one more time. Not using Github much. Do I have to re-create a new pull request or can I move this one to a new rev?

Reply:
Hm? Just add another commit to this branch with the fix, and push it.

Reply:
hmm wait. I don't see the type. are you looking at a prev revision?